_commands/tools/dc.md

```diff
@@ -2,22 +2,62 @@
 ---
 dc
 ----
-TODO: Add documentation for this command by submitting a pull request.
 <!-- one line explanation would go here -->
+dc is a desk calculator that supports unlimited-precision arithmetic and postfix notation.
 
 <!-- minimal example -->
 ~~~ bash
+dc
+2
+3
++
+p
+5
 *short* example of command usage and output
 ~~~
 
 <!--more-->
 
-### Useful Options / Examples
+### Note
+Because it is postfix notation, the operations come AFTER the numbers
+
+### Example
+You can add  complexity to it using multiple operators
+~~~ bash
+dc
+100
+0.5
+6.34
++
+*
+-
+p
+-684.00
+~~~
+
+### Useful Options
+
+#### Printing Commands
```

Comment:
Could you include examples of each of these printing commands? And also maybe include a command to exit, if there is one?